addons/project_timesheet_holidays/models/res_company.py

```diff
@@ -14,6 +15,13 @@ class Company(models.Model):
         'project.task', string="Leave Task",
         domain="[('project_id', '=', leave_timesheet_project_id)]")
 
+    @api.constrains('leave_timesheet_project_id')
+    def _check_leave_timesheet_project_id_company(self):
+        for company in self:
+            if company.leave_timesheet_project_id:
```

Comment:
`if company.leave_timesheet_project_id` is not necessary I think .

Reply:
In fact, for stable policy, I think I should delete the constrain. I will wait the opinion's devs.

Reply:
I don't think stable policy matters here.